scripts/ci-build.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+REPO_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
Why change this?

Reply:
If you look at failures here https://github.com/kubernetes-sigs/cluster-api/pull/4329, some of them are failing because that path is relative and scripts fails to get back to root folder from CAPD.

Reply:
I just copied the value from here https://github.com/kubernetes-sigs/cluster-api/blob/master/scripts/ci-e2e.sh#L20

Reply:
Sounds good, thanks!